CHANGELOG.md

```diff
@@ -26,9 +26,24 @@ Version template:
 
 ### Added
 
+* Added solr-telemetry [[#40]]
+	* Early version, without micrometer, Prometheus only
+
+	[#27]: https://github.com/xenit-eu/alfred-telemetry/pull/27
+
+	* With micrometer
+	
+	[#40]: https://github.com/xenit-eu/alfred-telemetry/pull/40 
+	[#43]: https://github.com/xenit-eu/alfred-telemetry/pull/43
+	[#44]: https://github.com/xenit-eu/alfred-telemetry/pull/44
+	[#46]: https://github.com/xenit-eu/alfred-telemetry/pull/46
+	[#47]: https://github.com/xenit-eu/alfred-telemetry/pull/47
+	
+
 * Added metrics tracker for NodeDAO and AclDAO (to get MaxTxNode )[[#42]]
 
 [#42]: https://github.com/xenit-eu/alfred-telemetry/pull/42
```

Comment:
```suggestion
* new module: `alfred-telemetry-solr` provides metrics for alfresco-solr4 and alfresco-solr6 [[#27][#40][#43][#44][#46][#47]]
* Added metrics tracker for NodeDAO and AclDAO (to get MaxTxNode )[[#42]]

[#40]: https://github.com/xenit-eu/alfred-telemetry/pull/40 
[#42]: https://github.com/xenit-eu/alfred-telemetry/pull/42
[#43]: https://github.com/xenit-eu/alfred-telemetry/pull/43
[#44]: https://github.com/xenit-eu/alfred-telemetry/pull/44
[#46]: https://github.com/xenit-eu/alfred-telemetry/pull/46
[#47]: https://github.com/xenit-eu/alfred-telemetry/pull/47
```

Reply:
compacter representation - not really useful to mention intermediate states in the changelog ?

Reply:
I wanted to keep a trace of that PR because it offers Prometheus monitoring without intervening in solr's core (micrometer version overwrites metrics-graphite from solr).
I will do that via this comment then :-)